.github/workflows/e2e.yml

```diff
@@ -105,4 +105,10 @@ jobs:
       - name: Cleanup
         if: always() && !inputs.skip-undeploy
         run: |
+          # Check if namespace file exists and is not empty
+          if [[ ! -f workspace/e2e.namespace ]]; then
+            echo "No namespace file found, skipping cleanup."
```

Comment:
This might obscure other issues with the run. Why do we need to run the test on runners that do not support it in the first place?

Reply:
I think there are two points here:
1. In general tests can be skipped. Should the cleanup error if no test have been run? In my opinion not.
2. The atls test is a general atls test but currently only has a snp implementation. So _in general_ the atls test can be run on TDX. It only contains one test function that skips the test on non-TDX.

Reply:
Ok, thanks